src/Console/Commands/MvcContractsUpdate.php

```diff
@@ -55,8 +55,8 @@ public function handle(DocumentSignInterface $documentSign)
                     $contract->status = Contract::STATUS_DENIED;
                     break;
                 case 'si':
-                    $contract->signed_at = Carbon::createFromTimestamp(strtotime($signRequest['signed_on']), 'Europe/Amsterdam');
                     $contract->status = Contract::STATUS_SIGNED;
```

Comment:
This is incorrect, the status should be SIGNED when all parties have signed. So you shoyld check the sign request response